### DISCUSSION TITLE
Add zeek after installation

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#5408] [Unsupported Versions] [answered]
Hi,
Is it possible to add zeek after the basic installation of Security Onion if Zeek was unselected?
Thank you

**Comment by dougburks** [comment, marked as answer]
Are you saying that when you ran Setup you selected Suricata as your metadata engine? Here are the steps to change the metadata engine from Zeek to Suricata and you should be able to do something similar to change Suricata to Zeek:
https://docs.securityonion.net/en/2.3/suricata.html#metadata

**Reply by yula-21** [reply]
I did that and after running so-status it showed that zeek is running but other checks we made returned the error: “no running instance of zeek”

I think that the problem is that not only was suricata chosen as the metadata engine but also zeek was unchecked when choosing the services to install on the initial setup.

**Reply by dougburks** [reply]
I'm not sure what part of setup you're referring to. If all else fails, you can always perform a fresh installation.

**Reply by yula-21** [reply]
Thank you for your replies, the problem is that we have all of our configuration set on the system already and running the setup again would reset everything as far as I know.
Do you have any idea why the error “no instances of zeek are running” when running so-zeek-status

**Reply by dougburks** [reply]
What is the output of the following?
```
sudo docker images |grep zeek
```
If no zeek images are found, you could try:
```
sudo so-docker-refresh
```

**Reply by yula-21** [reply]
It appears that zeek is running but for some reason logs aren't flowing to /nsm/zeek/logs

**Reply by dougburks** [reply]
You could always perform a fresh installation on another box (perhaps a VM) and then compare the two systems.
